### ISSUE TITLE
How reach good accuracy in Linear Evaluation

### ISSUE THREAD

**Issue by aynesss** [sayakpaul/SimCLR-in-TensorFlow-2#7]
In the notebook of pretrain, we must reach zero of contrastive Loss to have a good performance in Linear evaluation ?
Meaning that I must decrease loss to zero before passing to finetune ?

**Comment by sayakpaul** [comment]
It's good to have the contrastive loss to as low as possible without getting into representation collapse, yes. This repository provides a minimal implementation in order for developers to understand what's happening under the hood with SimCLR. I think I made this point clear in the README too. 

If you want to get good performance, I recommend using the official code of the method you are interested in. You can also try out the [`vissl`](https://vissl.ai/) library by FAIR that implements a wide range of different SSL methods.
